toolchain/check/import_cpp.cpp

```diff
@@ -131,10 +136,157 @@ auto ImportCppFiles(Context& context, llvm::StringRef importing_file_path,
   auto name_scope_id = AddNamespace(context, package_id, imports);
   SemIR::NameScope& name_scope = context.name_scopes().Get(name_scope_id);
   name_scope.set_is_closed_import(true);
+  name_scope.set_is_cpp_scope(true);
+
+  context.sem_ir().set_cpp_ast(generated_ast.get());
 
   if (ast_has_error) {
     name_scope.set_has_error();
   }
+
+  return std::move(generated_ast);
+}
+
+// Lookups the given name in the Clang AST. Returns the lookup result if lookup
```

Comment:
```suggestion
// Looks up the given name in the Clang AST. Returns the lookup result if lookup
```

Small grammar thing.

Reply:
Done.